lib/Twig/Extension/HelpersExtension.php

```diff
@@ -17,10 +17,12 @@
 
 namespace Pimcore\Twig\Extension;
 
+use Twig\Extension\AbstractExtension;
+
 /**
  * Simple helpers that do not need a dedicated extension
  */
-class HelpersExtension extends \Twig_Extension
+class HelpersExtension extends AbstractExtension
```

Comment:
Should not we also replace all deprecated \Twig_Test and \Twig_Function in all twig extensions?

Reply:
Sure these are deprecated? https://twig.symfony.com/doc/1.x/deprecated.html#tests https://twig.symfony.com/doc/1.x/deprecated.html#functions

Reply:
In the 2.x doc it is only marked as final:
https://twig.symfony.com/doc/2.x/deprecated.html

In the code the `\Twig_Test` and `\Twig_Function` throw deprecation warnings.
`\Twig\TwigTest` and `\Twig\TwigFunction` doesn't throws deprecation warnings.

I think the 1.x doc is out of date.

Reply:
I think it cames from search and replace over the doc: https://github.com/twigphp/Twig/commit/3632a46e82fbb391b0fc72674bd8946e0b7c5bf7#diff-dd6d4937e8b99ee5a7e9deb418f67b30L74

Reply:
yes, AFAIK we are not overriding these classes ;)

Reply:
I create a PR for the misleading doc: https://github.com/twigphp/Twig/pull/3001 :)